### ISSUE TITLE
Collection child with file input

### ISSUE THREAD

**Issue by realprabs** [jakub-zawislak/formex_ecto#4]
When I use file input fields in a collection, I get following error:

```
file_input/3 requires the enclosing form_for/4 to be configured with multipart: true
```

However, the enclosing form does have multipart true.

**Comment by jakub-zawislak** [comment]
I must admit that I have not even tested file uploads in collections 😄 
I will fix it soon, I know where the problem is

**Comment by jakub-zawislak** [comment]
Fixed in formex 0.6.1

**Comment by realprabs** [comment]
Thank you for resolving it so quickly!
